rules/Php81/Rector/Array_/FirstClassCallableRector.php

```diff
@@ -83,14 +90,57 @@ public function name()
      */
     public function getNodeTypes(): array
     {
-        return [Property::class, ClassConst::class, Array_::class];
+        return [Property::class, ClassConst::class, Array_::class, FuncCall::class];
     }
 
     /**
-     * @param Property|ClassConst|Array_ $node
+     * @param Property|ClassConst|Array_|FuncCall $node
      */
-    public function refactorWithScope(Node $node, Scope $scope): int|null|StaticCall|MethodCall
+    public function refactorWithScope(Node $node, Scope $scope): int|null|StaticCall|MethodCall|FuncCall
     {
+        if ($node instanceof FuncCall) {
+            if (! $node->name instanceof Name) {
+                return null;
+            }
+
+            if ($node->isFirstClassCallable()) {
+                return null;
+            }
+
+            $functionName = (string) $this->getName($node);
+
+            try {
+                $rf = new ReflectionFunction($functionName);
```

Comment:
I think PHPStan ReflectionProvider ->hasFunction() and ->getFunction() can be used

Reply:
I wanted to use the ReflectionProvider, however one of the implementations (\Rector\DowngradePhp80\Reflection\SimplePhpParameterReflection::getType) does not support getting the type for the parameter so I went back to native. What do you want me to do?

Reply:
not sure, we may can revisit later if only the native one is working